packages/perseus-score/src/empty-diner-check.ts

```diff
@@ -0,0 +1,72 @@
+import {getWidgetIdsFromContent} from "@khanacademy/perseus-core";
+
+import type {UserInputMap} from "./validation.types";
+import type {PerseusItem} from "@khanacademy/perseus-core";
+
+/**
+ * Check the emptiness of DINER widgets (for the AX team):
+ * - Dropdown
+ * - InteractiveGraph
+ * - NumericInput
+ * - Expression
+ * - Radio
+ *
+ * @param {PerseusItem} itemData
+ * @param {UserInputMap} userInputMap
+ * @returns {boolean} true if there's an empty widget, otherwise false
+ */
+function emptyDinerCheck(
```

Comment:
This function name doesn't immediately reveal the meaning of the answer. When going through the unit tests, I had to think about what `true` and `false` meant. What do you think of `hasEmptyDinerWidgets()`?  Or `hasEmptyWidgetsDiner()`?